GNUmakefile

```diff
@@ -2,19 +2,20 @@ TEST?=$$(go list ./... |grep -v 'vendor')
 GOFMT_FILES?=$$(find . -name '*.go' |grep -v vendor)
 WEBSITE_REPO=github.com/hashicorp/terraform-website
 PKG_NAME=fastly
+VERSION=$(shell git describe --tags --always)
 
 default: build
 
 build: fmtcheck
-	go install
+	go install -ldflags="-X github.com/terraform-providers/terraform-provider-fastly/version.ProviderVersion=$(VERSION)"
```

Comment:
I suggest moving the repo string "github.com/terraform-providers/terraform-provider-fastly/" to a variable at the top of the file to use in both of these lines.

Reply:
good point, let me fix that